python/cugraph/tests/dask/test_mg_bfs.py

```diff
@@ -12,26 +12,24 @@
 # limitations under the License.
 
 import cugraph.dask as dcg
-import cugraph.comms as Comms
-from dask.distributed import Client
 import gc
 import pytest
 import cugraph
 import dask_cudf
 import cudf
-from dask_cuda import LocalCUDACluster
-from cugraph.dask.common.mg_utils import is_single_gpu
+from cugraph.dask.common.mg_utils import (is_single_gpu,
+                                          setup_local_dask_cluster)
 
 
-@pytest.fixture
+@pytest.fixture(scope="module")
 def client_connection():
-    cluster = LocalCUDACluster()
-    client = Client(cluster)
-    Comms.initialize(p2p=True)
+    # setup
+    (comms, client, cluster) = setup_local_dask_cluster(p2p=True)
 
     yield client
 
-    Comms.destroy()
+    # teardown
```

Comment:
Maybe a teardown function similar to setup above could be added

Reply:
Thanks, I'll add this in the next commit.